UI/cmake/os-linux.cmake

```diff
@@ -1,5 +1,5 @@
 target_sources(obs-studio PRIVATE platform-x11.cpp)
-target_compile_definitions(obs-studio PRIVATE OBS_INSTALL_PREFIX="${OBS_INSTALL_PREFIX}")
+target_compile_definitions(obs-studio PRIVATE USE_XDG OBS_INSTALL_PREFIX="${OBS_INSTALL_PREFIX}")
```

Comment:
This adds the `USE_XDG` definition unconditionally, is that desired?

Reply:
You did the same in libobs.

Reply:
It is desired, since the code relies on defining USE_XDG.